templates/Web/Projects/AddPackageJsonFullStackDefault/.template.config/template.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": ["Universal"],
+  "name": "AddPackageJsonFullStackDefault",
+  "groupIdentity": "wts.Proj.AddPackageJsonFullStackDefault",
+  "identity": "wts.Proj.AddPackageJsonFullStackDefault",
+  "description": "",
+  "tags": {
+    "language": "Any",
+    "type": "project",
+    "wts.type": "project",
+    "wts.projecttype": "FullStackWebApp",
+    "wts.frontendframework": "all",
+    "wts.backendframework": "all",
+    "wts.platform": "Web",
+    "wts.version": "1.0.0",
+    "wts.displayOrder": "1"
+  },
+  "sourceName": "wts.AddPackageJsonFullStackDefault",
+  "preferNameDirectory": true,
+  "PrimaryOutputs": [],
+  "symbols": {
+  },
+  "postActions": [
+    {
+      "description": "Add Base Start Scripts",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
+      "args": {
+        "dict": "{'start': 'concurrently \"yarn start-frontend || npm run start-frontend\" \"yarn start-backend || npm run start-backend\"'}",
+        "key": "scripts",
+        "jsonPath": "package.json"
+      },
+      "continueOnError": false
+    }
+  ]
+}
```

Comment:
That's a good idea to make start script to another file that is independent of the front-end and back-end framework. I think it would also be nice to have the 'concurrently' dependency being added in this file because it would only be used when we run a full stack application.
Independently, the front-end and back-end frameworks should not required access to 'concurrently' package.

What do you think about it?

Reply:
Its possible if you want i can add it

Reply:
I think we should add it as it would make more sense for that package to be here :)

Reply:
We shall incorporate this in another PR.